phpstan/config/default.neon

```diff
@@ -0,0 +1,24 @@
+parameters:
+	level: 6
```

Comment:
I suggest we change this to `max` and then add a project-specific `level` in the `phpstan.neon` file. Like my comment below, if the company default config might possibly be extracted out into a Composer package, this will surely be set project to project.